scripts/test.sh

```diff
@@ -592,7 +592,7 @@ function release_pass {
   UPGRADE_VER=$(git ls-remote --tags https://github.com/etcd-io/etcd.git \
     | grep --only-matching --perl-regexp "(?<=v)${binary_major}.${previous_minor}.[\d]+?(?=[\^])" \
     | sort --numeric-sort --key 1.5 | tail -1 | sed 's/^/v/')
-  log_callout "Found latest release: ${UPGRADE_VER}."
+  log_callout "Found previous minor version (v${binary_major}.${previous_minor}) latest release: ${UPGRADE_VER}."
```

Comment:
Checking a few scenarios:

So for a `main` branch (i.e. `v3.7.0`) we see: https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/etcd-io_etcd/20127/pull-etcd-e2e-arm64/1930701976593502208#1:build-log.txt%3A692

```bash
 Found previous minor version (v3.6) latest release: v3.6.0. 
```

For a `release-3.6` run we see:

```bash
Found previous minor version (v3.5) latest release: v3.5.21.
```


For a `release-3.5` run we see:

```bash
Found previous minor version (v3.4) latest release: v3.4.37.
```